BigQuery/src/BigQueryClient.php

```diff
@@ -682,4 +682,22 @@ public function timestamp(\DateTimeInterface $value)
     {
         return new Timestamp($value);
     }
+
+    /**
+     * Get a service account for the KMS integration.
+     *
+     * Example:
+     * ```
+     * $serviceAccount = $bigQuery->getServiceAccount();
+     * ```
+     * @return string
+     */
+    public function getServiceAccount()
+    {
+        $resp = $this->connection->getServiceAccount(
```

Comment:
It looks like the indentation on the closing brace is causing CI to fail. WDYT about this formatting?

```php
$resp = $this->connection->getServiceAccount([
    'projectId' => $this->projectId
]);

return $resp['email'];
```